### DISCUSSION TITLE
State of HTML 2023 Impact

### DISCUSSION THREAD

**Discussion by LeaVerou** [Devographics/surveys#223] [State of HTML General]
Tracking discussion, which will be updated with links as they happen.

- Interop 2024
	- [Oct 24 Presentation](https://docs.google.com/presentation/d/1AVsalCAZVITkYZCLxux5D2LbwLukjjuH9cm-vZRYy-w/edit#slide=id.p)
	- [Preliminary results shared with group](https://github.com/web-platform-tests/interop/issues/550#issuecomment-1777758744)
	- https://github.com/web-platform-tests/interop/issues/423#issuecomment-1779952138
	- https://github.com/web-platform-tests/interop/issues/537#issuecomment-1780208403
	- https://github.com/web-platform-tests/interop/issues/572#issuecomment-1778590205
	- https://github.com/web-platform-tests/interop/issues/523#issuecomment-1778590889
	- https://github.com/web-platform-tests/interop/issues/553#issuecomment-1778591890
	- https://github.com/web-platform-tests/interop/issues/464#issuecomment-1778592261
- Open UI
	- https://github.com/openui/open-ui/issues/903

**Comment by SachaG** [comment]
That's great, thanks for tracking this.
